src/scheduler.rs

```diff
@@ -96,6 +96,9 @@ async fn inbox_loop(ctx: Context, started: Sender<()>, inbox_handlers: ImapConne
                     info = if ctx.get_config_bool(Config::InboxWatch).await {
                         fetch_idle(&ctx, &mut connection, Config::ConfiguredInboxFolder).await
                     } else {
+                        if let Err(err) = connection.scan_folders(&ctx).await {
```

Comment:
Good catch!

Reply:
Maybe add a python test for it, though?

Reply:
I don't know... We are planning to remove the InboxWatch option, anyway (as well as the other watch options), and the python test would test "If InboxWatch is disabled, scan_folders() should still work"

Reply:
Issue for watch folder setting is #2140, for reference